scripts/test-go.sh

```diff
@@ -21,27 +25,41 @@ else
     exit 1
 fi
 
-if [ "${SKIP_NON_GENERATED_FILES}" = false ]; then
-    echo ">> Testing core"
-    cd ${CORE_PATH}
-    go test ./... ${GOTEST_ARGS}
-fi
-
-for service_dir in ${SERVICES_PATH}/*; do
-    service=$(basename ${service_dir})
-
-    # Our unit test template fails because it doesn't support fields with validations,
-    # such as the UUID component used by IaaS. We introduce this hardcoded skip until we fix it
-    if [ "${service}" = "iaas" ] || [ "${service}" = "iaasalpha" ]; then
-        echo ">> Skipping services/${service}"
-        continue
-    fi
-
-    echo ">> Testing services/${service}"
-    cd ${service_dir}
+# If a service is specified, only test that service
+if [ ! -z "${SERVICE}" ]; then
+    echo ">> Testing services/${SERVICE}"
+    cd ${SERVICES_PATH}/${SERVICE}
```

Comment:
I guess this was already a thing before, but we don't actually need to `cd` into the folders, we can just run `go test ${SERVICES_PATH}/${SERVICE}/... ${GOTEST_ARGS}`